package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "vease",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Basic app",
-  "main": "electron/main.js",
+  "main": "dist-electron/main.js",
```

Comment:
Dist electron should be committed or not?

Reply:
In my opinion it shouldn't because it is generated, but this says to the build package where the entry point is